### PR TITLE
Add script to generate ticker (rocket replacement) files

### PR DESCRIPTION
**Pull request by tshastry**
I will try to fix codeclimate issues as they come up.

### CODE REVIEW

File: util/generate_ticker_file.py [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+
+    return seen_entities, entity_working_list
+
+
+def create_unique_minimum_entities(entities):
+    """
+    Retuns an entity list where each entity is as short
+    as possible while maintaining a unique list
+  
+    >>> create_unique_minimum_entities(['abc one company', 'abc two company', 'def three company'])
```
Comment (mgoldey) [inline util/generate_ticker_file.py]: This behavior may be a problem if a client wants to add "just one more" and doesn't use the whole list. We'll need to coach clients through this one.

**Comment by tshastry** [comment]
The intended use case here is that if you're using it for chat, you use it as such:

1) Create a companies.csv file with two columns for company name and ticker:

```
company name, ticker
corp one, ABC
corp two, DEV
three, GHI
```

2) Generate a company -> ticker mapping:

```
python util/generate_ticker_list.py companies.csv
```

and you get a file like this called `companies.txt`:

```
corp two -> DEV
corp one -> ABC
three -> GHI
```

3) Generate a ticker -> ticker mapping:

```
python util/generate_ticker_file.py companies.csv --tickers_only
```

and you'll get a file like this called `companies_tickers.txt`:

```
d e v -> DEV
a b c -> ABC
g h i -> GHI
```

4) If you wish to use both files for entity definitions, simply combine them.

```
cat companies_tickers.txt >> companies.txt
```

File: util/generate_ticker_file.py [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+
+
+def clean_entity(entity):
+    """
+    Clean formatting from an entity
+  
+    >>> clean_entity('Test')
+    'test'
+    >>> clean_entity('ABC Holdings')
+    'abc holdings'
```
[the remark this answers is not in the document]
Comment (tshastry) [reply util/generate_ticker_file.py]: See the comment below

File: util/generate_ticker_file.py [hunk 3]
```diff
@@ -0,0 +1,173 @@
...
+    >>> tickers_to_rockets([['Test','TST']])
+    ['t s t -> TST']
+    """
+    tickers = [e[1].strip() for e in entity_pairs]
+    entities = [' '.join(list(e.lower())) for e in tickers]
+
+    return ["{0} -> {1}".format(x, y) for x, y in zip(entities, tickers)]
+
+
+def generate_ticker_file(filename,
```
Comment (mgoldey) [inline util/generate_ticker_file.py]:
```suggestion
def generate_ticker_file(filename,|
  """
  * Create a companies.csv file with two columns for company name and ticker: company name, ticker
  corp one, ABC
  corp two, DEV
  three, GHI
  
  * Generate a company -> ticker mapping:
  python util/generate_ticker_list.py companies.csv
  and you get a file like this called companies.txt:

  corp two -> DEV
  corp one -> ABC
  three -> GHI
  
  * Generate a ticker -> ticker mapping:
  python util/generate_ticker_file.py companies.csv --tickers_only and you'll get a file like this called companies_tickers.txt:

  d e v -> DEV
  a b c -> ABC
  g h i -> GHI
  
  If you wish to use both files for entity definitions, simply combine them.
  cat companies_tickers.txt >> companies.txt
  """
```

Comment (tshastry) [reply util/generate_ticker_file.py]: Thanks -- this will go in the wiki too once merged.

**Pull request merged**